packages/headless/src/controllers/search-box/headless-search-box-options.ts

```diff
@@ -35,16 +35,24 @@ export interface SearchBoxOptions {
    * @defaultValue `5`
    */
   numberOfSuggestions?: number;
+
+  /**
+   * Specifies whether to clear filters when the user enters a new search query
```

Comment:
I like the other description better: `Whether the filters should be cleared upon a new search.`